syntax = "proto2";

message s2c_player_attribute_notify {
	required string name = 1;		// 玩家名字
	required uint32 face = 2;		// 头像
	required uint32 lv = 3;			// 等级
	required uint64 exp = 4;		// 经验
	required uint64 gold = 5;		// 金币
	required uint64 money = 6;		// 砖石
	required uint64 vitality = 7;	// 体力
}